### PR TITLE
Time Between Data Output

### PR DESCRIPTION
**Pull request by TylerLaBree**
This is a feature requested on the Projects tab. Now, instead of creating a set number of snapshots equally spaced between sim start and end_time, it instead takes snapshots at a set time interval, so we should be able to make better animations with them.

I also fixed a bug. I know it's not a good idea to do more than one thing in a pull request, but I'm not sure how to undo this fix so I can fix it in another branch. Does anyone have any thoughts on this?

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -62,12 +62,13 @@ int main(int argc, char* argv[]){
         InputFileWithTimeInterval<PTCL::RealPtcl>(sph_system, sysinfo);
         PROBLEM::setEoS(sph_system);
     }
+    PS::F64 output_interval = parameter_file.getValueOf("output_interval",100.);
```
Comment (gassmoeller) [inline src/main.cpp]: Does it make sense to change the default value to the value the output_interval was before? I.e. the default end time is 1e4, and the old number of snapshots was set to 200, so maybe make 50 the default output interval? Or is 100 preferable in some way?

File: src/io.h [hunk 2]
```diff
@@ -37,7 +37,7 @@ template <class ThisPtcl> void OutputFileWithTimeInterval(PS::ParticleSystem<Thi
             std::cout << "output " << full_filename << "." << std::endl;
 			std::cout << "//================================" << std::endl;
 		}
-		sysinfo.output_time += end_time / PARAM::NUMBER_OF_SNAPSHOTS;
+        sysinfo.output_time += output_interval;
```
Comment (gassmoeller) [inline src/io.h]: While we have not decided on #26 can  you replace these spaces by two tabs? At least keep it consistent where possible.

**Review by gassmoeller — COMMENTED** [review]
Looks good, just two minor comments.

File: src/main.cpp (hunk 1 above)
Comment (TylerLaBree) [reply src/main.cpp]: That's an excellent idea. I didn't think about calculating what it was before by default. I'll implement this.

**Comment by TylerLaBree** [comment]
I should have the changes you requested fixed now @gassmoeller. Thanks for the feedback.

**Comment by gassmoeller** [comment]
Thanks! Ready to merge.

**Pull request merged**